charts/tidb-operator/values.yaml

```diff
@@ -153,6 +153,9 @@ scheduler:
   # runAsUser: 1000
   # runAsGroup: 2000
   # fsGroup: 2000
+  podAnnotations: {}
+  # prometheus.io/scrape: "true"
+  # prometheus.io/port: "9090"
```

Comment:
```suggestion
  # Refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
  annotations: {}
```

Reply:
Suggest also adding the annotations for the `controller-manager` section.

Reply:
@max-f-z I just revert the parameter name, the original name is good.

Reply:
ok, will add annotations for controller-manager as well, thanks @DanielZhangQD